scripts/update-dependencies/Config.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
nit: aspnetcore source code typically uses the apache license.
https://github.com/aspnet/Universe/blob/c213def07af73590c5f3df0b68f60fc8b66682ea/build/tasks/AddArchiveReferences.cs#L1-L2